FVCalc.java

```diff
@@ -1,6 +1,11 @@
 // Computes the future value of a saving investment.
 public class FVCalc {
 	public static void main(String[] args){
-		// Replace this comment with your code
+		int sum = Integer.parseInt(args[0]);
+		double percent = Double.parseDouble(args[1]);
+		int yrs = Integer.parseInt(args[2]);
+		double futureVal = sum * Math.pow(percent/100.0 + 1, yrs);
```

Comment:
spacing, I would also add parenthesis for making it clearer 
`Math.pow((percent / 100.0) + 1, yrs);`